scene/main/viewport.h

```diff
@@ -480,6 +480,7 @@ class Viewport : public Node {
 	void _validate_property(PropertyInfo &p_property) const;
 
 public:
+	void force_can_drop_data_check();
```

Comment:
I would move this down next to other dragging related methods like `gui_cancel_drag`.
Same with the implementation.